modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -53,17 +53,54 @@ spec:
 <3> Last observed state of the catalog connection. The `READY` value ensures that the `CatalogSource` policy is ready, indicating that the index pod is pulled and is running. This way, {cgu-operator} upgrades the Operators based on up-to-date policy compliance states.
++
+[NOTE]
+====
+After a catalog source update, it takes time for OLM to update the subscription status. This can mean that the status of the subscription policy may continue to show as compliant when TALM decides whether remediation is needed. As a result the Operator specified in the subscription policy does not get upgraded. To mitigate this issue, the `lastObservedState: READY` specification reduces the time between the completion of the catalog source policy remediation and the update of the subscription status. 
+
+However, if the issue persists, you can add another catalog source configuration to the `PolicyGenTemplate` resource with the same specifications, except for the following changes: 
+
+[source,yaml]
+----
+- fileName: DefaultCatsrc.yaml
+  # Contains original catalog source configuration
+  ...
+- fileName: DefaultCatsrc.yaml 
+      remediationAction: inform
+      policyName: "operator-catsrc-policy"
+      metadata:
+        name: redhat-operators-v2 <1>
+      spec:
+        displayName: Red Hat Operators Catalog v2 <2>
+        image: registry.example.com:5000/olredhat-operators:<version> <3>
+...
+----
+<1> Update the name for the new configuration.
+<2> Update the display name for the new configuration.
+<3> Update the index image URL. This `fileName.spec.image` field overrides any configuration in the `DefaultCatsrc.yaml` file.
+
+Update the `Subscription` resource to point to the new configuration for Operators that require an update:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+spec:
+  source: redhat-operators-v2 <1>
+----
+<1> Enter the name of the additional catalog source configuration that you defined in the `PolicyGenTemplate` resource.
+====
 
 .. This update generates one policy, `du-upgrade-operator-catsrc-policy`, to update the `redhat-operators` catalog source with the new index images that contain the desired Operators images.
-+
+
 [NOTE]
 ====
 If you want to use the image pre-caching for Operators and there are Operators from a different catalog source other than `redhat-operators`,  you must perform the following tasks:
 
-* Prepare a separate catalog source policy with the new index image or registry poll interval update for the different catalog source.
+* Prepare a separate catalog source policy with the new index image or registry poll interval update for the different catalog source. 
```

Comment:
I've had to read "for the different catalog source." a few times. Can this be dropped or maybe specific:

> you must perform the following tasks for the different catalog source:

Differently configured?

Reply:
This line is whitespace trimming only, not something that was modified in the PR